nimble/host/include/host/ble_gap.h

```diff
@@ -428,6 +429,17 @@ struct ble_gap_disc_desc {
     ble_addr_t direct_addr;
 };
 
+struct ble_gap_pairing_req {
+    uint16_t conn_handle;
+    /** Properties of the existing pairing request */
+    uint8_t io_cap;
+    uint8_t oob_data_flag;
+    uint8_t authreq;
+    uint8_t max_enc_key_size;
+    uint8_t init_key_dist;
+    uint8_t resp_key_dist;
+};
```

Comment:
I don't think we should just expose to app anything that is in SMP request... what app could use key distribution settings for?

Reply:
Totally agree to that.. will make the change !!

Reply:
Done !!